contrib/babelfishpg_tsql/Makefile

```diff
@@ -94,15 +94,15 @@ UPGRADES = $(patsubst sql/upgrades/%.sql,sql/%.sql,$(wildcard sql/upgrades/*.sql
 REGRESS = test/babel_like
 REGRESS += test/babel_219
 REGRESS += test/babel_init
-REGRESS += test/babel_select_distinct_top
+# REGRESS += test/babel_select_distinct_top
```

Comment:
why is this commented?

Reply:
The test-file with that name does not exist

Reply:
It exist in internal so I think I need not comment it. I will revert it